sdk/synapse/Microsoft.Azure.Management.Synapse/src/Microsoft.Azure.Management.Synapse.csproj

```diff
@@ -7,11 +7,14 @@
 		<PackageId>Microsoft.Azure.Management.Synapse</PackageId>
 		<Description>Azure Synapse Management SDK Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Synapse</AssemblyName>
-		<Version>0.1.0-preview.1</Version>
+		<Version>0.2.0-preview</Version>
```

Comment:
Is 0.2.0-preview right here?  Should it be `0.1.0-preview.2` or `0.2.0-preview`?
This PR adds some api.

Reply:
`0.1.0-preview.2`. And when you decide it's stable, `0.1.0`.
And then `0.2.0-preview.1`, and then `0.2.0`

Reply:
> Is 0.2.0-preview right here? Should the version be `0.1.0-preview.2` or `0.2.0-preview`?
> If the version is 0.2.0-preview, can we set the version to 0.1.0 when we finish the preview phase?

Thanks for clarification. I have updated it to 0.1.0-preview.2.